src/Api/ProjectsApi.php

```diff
@@ -86,7 +86,28 @@ public function projectsFeaturedGet(string $platform = null, string $maxVersion
    */
   public function projectsGet(string $project_type, ?string $accept_language = null, ?string $max_version = null, ?int $limit = 20, ?int $offset = 0, ?string $flavor = null, &$responseCode, array &$responseHeaders)
   {
-    // TODO: Implement projectsGet() method.
+    if (null == $max_version)
```

Comment:
It would be better to use strict comparisons.
https://stackoverflow.com/questions/80646/how-do-the-php-equality-double-equals-and-identity-triple-equals-comp